test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,445 @@
+          text: '-1.5 TST',
+        });
+        await driver.waitForSelector({
+          css: '.list-item__heading',
+          text: 'Send TST',
+        });
+        await driver.clickElement({
+          text: 'Assets',
+          tag: 'button',
+        });
+        const assets = await driver.findElement('[title="8.5 TST"]');
+        assert.ok(assets, 'Token amount is not correct');
+      },
+    );
+  });
+});
+
+describe('Transfers a custom token from dapp when no gas value is specified', function () {
+  const smartContract = SMART_CONTRACTS.HST;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  it('transfers an already created token without specifying gas, submits the transaction, finds the transaction in the transactions list', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'connected-state',
+        ganacheOptions,
+        smartContract,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver, contractRegistry }) => {
+        const contractAddress = await contractRegistry.getContractAddress(
+          smartContract,
+        );
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // create token
+        await driver.openNewPage(
+          `http://127.0.0.1:8080/?contract=${contractAddress}`,
+        );
+        let windowHandles = await driver.getAllWindowHandles();
+        const extension = windowHandles[0];
+        await driver.findClickableElement('#deployButton');
+        // add token from dapp
+        await driver.clickElement({
+          text: 'Add Token to Wallet',
+          tag: 'button',
+        });
+
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.clickElement({ text: 'Add token', tag: 'button' });
+        windowHandles = await getWindowHandles(driver, 2);
+        // transfer token without gas from dapp
+        await driver.clickElement({
+          text: 'Transfer Tokens Without Gas',
+          tag: 'button',
+        });
+
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.waitForSelector({
+          tag: 'h1',
+          text: '1.5 TST',
+        });
+        await driver.waitForSelector({ text: 'Confirm', tag: 'button' });
```

Comment:
I think we can remove this line

Reply:
Same as above. If I remove it test fails since it needs more time to load popup.

Reply:
Thank you, that make sense! Let's leave it as it is then! @mirjanaKukic